services/frontendhost/frontend.go

```diff
@@ -303,13 +303,38 @@ func convertCGZoneConfigToInternal(cgZoneCfg *c.ConsumerGroupZoneConfig) *shared
 
 // convertCreateCGRequestToInternal converts Cherami CreateConsumerGroupRequest to internal shared CreateConsumerGroupRequest
 func convertCreateCGRequestToInternal(createRequest *c.CreateConsumerGroupRequest) *shared.CreateConsumerGroupRequest {
+
+	// detect and correct the units for 'startFrom' (expected internally to be in nanoseconds)
+	startFrom := createRequest.GetStartFrom()
+
+	// Note: 2000-01-01T00:00 is 946684800 seconds since epoch
+	switch {
+	case startFrom == 0:
+		// special-case '0' and let it through
+
+	case startFrom < 946684800000:
```

Comment:
I verified the behaviour with CLI. If *no* start-time is specified, it uses a StartFrom of "now". But if the user explicitly specifies "0", the CLI passed down "0".

I think we should fix our documentation to not say "1", but say "0" instead.